ui/app/adt/controllers/wardListController.js

```diff
@@ -9,6 +9,23 @@ angular.module('bahmni.adt')
                 $window.location = appService.getAppDescriptor().formatUrl(Bahmni.ADT.Constants.ipdDashboard, options, true);
             };
 
+            $scope.searchText = '';
+                        
+            $scope.searchTextFilter = function (row) {
+                var searchText = $scope.searchText;
+                if (!searchText) {
+                    return true;
+                }
+                searchText = searchText.toLowerCase();
+                const excludedKeys = ["hiddenAttributes", "$$hashKey", "Diagnosis"];
+                const attributes = Object.keys(row).filter(key => !excludedKeys.includes(key));
```

Comment:
Minor suggestion. The attributes are essentially static and wont change. So having all the attributes in an array outside of this search function and not using Object.keys() here would result in that keys() operation not being repeated per call. Ofcourse, the .some() operation needs to be here.

Reply:
The attributes are not static. These attributes keep changing depending on what config we are using. Example:

Attributes on my local due to some config i'm using:

Bed | Identifier | Name | Age | Gender | District | Village | Admission By | Admission Time | Disposition By | Disposition Time | ADT Notes | Diagnosis

Attributes on our IPD QA env:

Bed | Ward | Identifier | Name | Age | Gender  | Admission Time | Disposition By | Disposition Time | ADT Notes | Diagnosis


Bed | Ward | Identifier | Name | Age | Gender | Admission Time | Disposition By | Disposition Time | ADT Notes | Diagnosis
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --




Bed | Ward | Identifier | Name | Age | Gender | Admission Time | Disposition By | Disposition Time | ADT Notes | Diagnosis
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --




Bed | Ward | Identifier | Name | Age | Gender | Admission Time | Disposition By | Disposition Time | ADT Notes | Diagnosis
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --




Bed | Ward | Identifier | Name | Age | Gender | Admission Time | Disposition By | Disposition Time | ADT Notes | Diagnosis
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --




Bed | Ward | Identifier | Name | Age | Gender | Admission Time | Disposition By | Disposition Time | ADT Notes | Diagnosis
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --